deal.II-toolchain/platforms/supported/sierra.platform

```diff
@@ -0,0 +1,33 @@
+# macOS 10.12 (Sierra)
+#
+# macOS support is experimental and requires:
+# - a recent version of xcode with command line tools
+# - openmpi installed using homebrew and
+# - cmake
+# 
+# Detailed Installation Notes:
+# - install Xcode from AppStore, open it and accept the license.
+# - Open Terminal, and install Xcode command line tools via
+# $ xcode-select --install
+# and then run
+# $ xcodebuild -license
+# to accept the license.
+# 
+# Install Homebrew in a Terminal via
+# $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+# 
+# Install the following via Homebrew
+# $ brew install cmake
+# $ brew install openmpi --c++11 --with-mpi-thread-multiple
+# 
+# (Optional) Install the following via Homebrew
+# $ brew install wget
+# $ brew install gnuplot
+# 
+# $ brew install bash
```

Comment:
Yes, but it shows an error for parsing if statements with double brackets [[ ]]. We use them to switch on / off packages.